### PR TITLE
Improve USAGE to make it more useful

### PR DESCRIPTION
**Pull request by elmetal**
Added some usage examples to README.md.

### CODE REVIEW

**Comment by elmetal** [comment]
Should the USAGE section be moved to the Early Part?

File: README.md [hunk 1]
```diff
@@ -72,6 +72,23 @@ WebUI is available through [Swift Package Manager](https://github.com/apple/swif
...
+```swift
+struct ContentView: View {
+    var body: some View {
+        WebView(request: URLRequest(url: URL(string: "https://example.com/")!))
+    }
+}
+```
+
+### Manipulating WebView
+Use `WebViewReader`.Within the scope of `WebViewReader`, you can receive `WebViewProxy`.You can manipulate `WebView` within the scope of `WebViewReader` via `WebViewProxy`.
```
Comment (Kyome22) [inline README.md:91]:
```suggestion
Use `WebViewReader`.Within the scope of `WebViewReader`, you can receive `WebViewProxy`.
You can manipulate `WebView` within the scope of `WebViewReader` via `WebViewProxy`.
```

**Comment by Kyome22** [comment]
> Should the USAGE section be moved to the Early Part?

Do you like the order of the chapters below?

1. Requirements
2. Usage
3. Documentation
4. Installation
5. Privacy Manifest
6. Demo

**Comment by elmetal** [comment]
> Do you like the order of the chapters below?
> 
> 1. Requirements
> 2. Usage
> 3. Documentation
> 4. Installation
> 5. Privacy Manifest
> 6. Demo

Good order!

**Review by Kyome22 — APPROVED** [review]

**Pull request merged**